src/mmap_disk_io.cpp

```diff
@@ -1374,11 +1378,16 @@ TORRENT_EXPORT std::unique_ptr<disk_interface> mmap_disk_io_constructor(
 		return false;
 	}
 
-	void mmap_disk_io::thread_fun(job_queue& queue
-		, disk_io_thread_pool& pool)
+	void mmap_disk_io::thread_fun(job_queue& queue, disk_io_thread_pool& pool)
 	{
 		std::thread::id const thread_id = std::this_thread::get_id();
 
+#ifdef _WIN32
+		MEMORY_PRIORITY_INFORMATION const info{MEMORY_PRIORITY_BELOW_NORMAL};
```

Comment:
This struct is only available on Windows 8 and later, so you might want to add some guard here.
https://docs.microsoft.com/en-us/windows/win32/api/processthreadsapi/ns-processthreadsapi-memory_priority_information

Reply:
Windows 7 was recently EOL'd (yes, I realize this does not magically make it disappear). Not sure what libtorrent's support policy is, but I think this is worth mentioning.

Reply:
I should probably use the helpers I already have to load the shared library at run-time and detect whether this function exists or not. `get_library_procedure()` in `win_util.hpp`